src/engraving/style/styledef.h

```diff
@@ -123,12 +123,12 @@ enum class Sid {
     lyricsOddFontStyle,
     lyricsOddColor,
     lyricsOddAlign,
-    lyricsOddFrameType,
-    lyricsOddFramePadding,
-    lyricsOddFrameWidth,
-    lyricsOddFrameRound,
-    lyricsOddFrameFgColor,
-    lyricsOddFrameBgColor,
+    lyricsOddBorderType,
+    lyricsOddBorderPadding,
+    lyricsOddBorderWidth,
+    lyricsOddBorderRound,
+    lyricsOddBorderFgColor,
+    lyricsOddBorderBgColor,

```

Comment:
The fact that these style settings are renamed is good, but you'll need to add compatibility handling so that existing files are still read correctly. See `MStyle::read`.

Reply:
Should I just add more else-if statements to check for the other version of the tags in this function?